package.json

```diff
@@ -5,26 +5,40 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@commitlint/cli": "^7.5.2",
+    "@commitlint/config-conventional": "^7.5.0",
     "eslint": "^5.13.0",
     "eslint-config-prettier": "^4.0.0",
     "eslint-loader": "^2.1.2",
     "eslint-plugin-flowtype": "^3.4.1",
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-react": "^7.12.4",
+    "husky": "^1.3.1",
     "lerna": "^3.4.3",
     "lint-staged": "^8.1.3",
     "prettier": "^1.16.4"
   },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
   "husky": {
     "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
       "pre-commit": "lint-staged"
     }
   },
+  "prettier": {
+    "useTabs": true,
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
   "lint-staged": {
-    "{test,js}/**/*.js": [
-      "prettier --write --single-quote --use-tabs --trailing-comma es5",
+    ".jsx": [
```

Comment:
We do have a couple of `.js` files in the `utils/` directory, it'd be goo to add linting back in for both .js + .jsx